core/src/checks/check_sign_tx.c

```diff
@@ -126,6 +133,62 @@ int verify_sign_tx(void) {
     print_bytes(resp.signatures[0].der.bytes, resp.signatures[0].der.size);
     return -1;
   }
+  
   INFO("sign_tx passed");
   return 0;
 }
+/**
+ * This function does not make an effort to zeroize
+ * anything as all of the values are test values.
+ * It will just test our verify signature wrapper.
+ */
+
+int verify_check_signature_pub(void){
+  //random ECDSA private key.
+  unsigned char private_key[32] = {
```

Comment:
super nit: we're using `uint8_t` types elsewhere, would be good to be consistent